Assets/Scripts/SS3D/Systems/Tile/TileMapCreator/TileMapCreator.cs

```diff
@@ -1,407 +1,473 @@
 using Coimbra;
```

Comment:
Did you do any other changes apart the namespace change in this file ? It's hard to check given the diff being so large.

Reply:
I don't think so, not sure why the diff is that gigantic